objax/io/checkpoint.py

```diff
@@ -109,6 +111,8 @@ def save(self, vc: VarCollection, idx: int):
             vc: variables collection to save.
             idx: index of the new checkpoint where variables should be saved.
         """
+        assert isinstance(vc, VarCollection), \
+            f"Must pass a VarCollection to save; received type {type(vc)}."
```

Comment:
```suggestion
        assert isinstance(vc, VarCollection), f'Must pass a VarCollection to save; received type {type(vc)}.'
```
Nits: (1) use single quotes and (2) we allow 120 character lines in code.